@uportal/notification-modal/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@uportal/notification-modal",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
`private` should be replaced by something similar to https://github.com/uPortal-contrib/uPortal-web-components/blob/835cdca41931ec0d78e68161ea3b29ad2f904d15/%40uportal/user-profile-menu/package.json#L4-L16